index.html

```diff
@@ -1,154 +1,192 @@
-        <img src="./images/catering.png"   class="man"   alt="" >
-      </div>
-      <div class="flip-card-back">
-        <h1>Catering</h1> 
-        <p>Catering Services</p> 
-        <p>We at Dev-Kitchen also offers catering service.For more information contact us.</p>
+    <div class="flip-card">
+      <div class="flip-card-inner">
+        <div class="flip-card-front">
+          <img src="./images/catering.png" class="man" alt="">
+        </div>
+        <div class="flip-card-back">
+          <h1>Catering</h1>
+          <p>Catering Services</p>
+          <p>We at Dev-Kitchen also offers catering service.For more information contact us.</p>
+        </div>
       </div>
     </div>
-  </div>
 
 
 
 
   </main>
-    <!-- Footer -->
-	<footer>
-		<div class="container" style="padding: 50px 0;" align="center">
-			<a href="https://www.facebook.com/DevIncept">
-				<i class="fa fa-facebook fa-lg white-text mr-md-5 mr-3 fa-2x"></i>
-			</a>
-			<a href="mailto:support@devincept.tech">
-				<i class="fa fa-google-plus fa-lg white-text mr-md-5 mr-3 fa-2x"></i>
-			</a>
-			<a href="https://www.linkedin.com/company/devincept/mycompany/">
-				<i class="fa fa-linkedin fa-lg white-text mr-md-5 mr-3 fa-2x"></i>
-			</a>
-			<a href="https://www.instagram.com/devincept.tech/">
-				<i class="fa fa-instagram fa-lg white-text mr-md-5 mr-3 fa-2x"></i>
-			</a>
-			<a href="#">
-				<i class="fa fa-twitter fa-lg white-text mr-md-5 mr-3 fa-2x"></i>
-			</a>
-		</div>
-		<div style="background: #1a1d1f;">
-			<p style="color: #b0f8ff; padding: 15px 0;" align="center">
-			&copy; 2021 Copyright: <a href="mailto:devincept@gmail.com"><b>Created By: DevIncept</b></a>
-			</p>
-		</div>
-	</footer>
-
+  <!-- Footer -->
+  <footer>
+    <div class="container" style="padding: 50px 0;" align="center">
+      <a href="https://www.facebook.com/DevIncept">
+        <i class="fa fa-facebook fa-lg white-text mr-md-5 mr-3 fa-2x"></i>
+      </a>
+      <a href="mailto:support@devincept.tech">
+        <i class="fa fa-google-plus fa-lg white-text mr-md-5 mr-3 fa-2x"></i>
+      </a>
+      <a href="https://www.linkedin.com/company/devincept/mycompany/">
+        <i class="fa fa-linkedin fa-lg white-text mr-md-5 mr-3 fa-2x"></i>
+      </a>
+      <a href="https://www.instagram.com/devincept.tech/">
+        <i class="fa fa-instagram fa-lg white-text mr-md-5 mr-3 fa-2x"></i>
+      </a>
+      <a href="#">
+        <i class="fa fa-twitter fa-lg white-text mr-md-5 mr-3 fa-2x"></i>
+      </a>
+    </div>
+    <div style="background: #1a1d1f;">
+      <p style="color: #b0f8ff; padding: 15px 0;" align="center">
+        &copy; 2021 Copyright: <a href="mailto:devincept@gmail.com"><b>Created By: DevIncept</b></a>
+      </p>
+    </div>
+  </footer>
+
+  <!-- scroll to top button -->
+  <button id="btnScrollToTop">
+    <img src="images/scrtotop.jfif" alt="scrbtn">
+  </button>
+  
+  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>
```

Comment:
You missed **https:** in this line, write like this  
```<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>```
That's why it wasn't working.

Reply:
oh sorry, my bad. Added